bundles/org.openhab.binding.lgthinq/src/main/resources/OH-INF/thing/dish-washer.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lgthinq"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- LG Device Type for DishWasher -->
+	<thing-type id="204">
```

Comment:
This is because each LG device type has an ID that uniquely identify it. Then I use this ID (thats numeric) only to easily correlate to the LG ID of the device. But I can change it if there is some specific convention. To be honest, I was not caring about this ID since it's just an... unique ID. Let me know please.

Reply:
You could add a XML comment like:  `<!-- api device id: 204 -->` as a sort of note to yourself or other developers.
This is not _just_ an unique id, users will see it in the UI and will also configure it manually when using file based configuration. So i would prefer to remove the numbers.

Reply:
Fixed. The composite <device-type-name>-<lg-deviceId> is enought to guide developer. See if it's ok right now.